doc/Repair_auxdb.rst

```diff
@@ -0,0 +1,118 @@
+============
```

Comment:
In general, this document is out-dated.
But we can use this an example for document conversion.

Reply:
Okay